hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/RMAppManager.java

```diff
@@ -640,6 +643,12 @@ private RMAppImpl createAndPopulateNewRMApp(
           SchedulerUtils.normalizeAndValidateRequest(amReq, maxAllocation,
               queue, isRecovery, rmContext, null, nodeLabelsEnabled);
 
+          String amNodeLabel = amReq.getNodeLabelExpression();
+          if (org.apache.commons.lang3.StringUtils.isEmpty(amNodeLabel)
```

Comment:
Please use a simpler package identifier ie. Stringutils.isEmpty, or import isEmpty directly.

Reply:
Got it. Thanks